events/health_events.txt

```diff
@@ -10767,21 +10767,9 @@ health.7000 = {
 	theme = physical_health
 
 	trigger = {
-		NOT = { has_trait = infirm }
-		OR = {
-			# Warcraft
-			age >= age_50_value
-
-			AND = {
-				# Warcraft
-				age >= age_30_value
-
-				OR = {
-					has_trait = physique_bad
-					has_trait = spindly
-					has_trait = weak
-				}
-			}
+		NOT = { 
+			has_trait = infirm 
+			age_ranked_health_vulnerability_threshold_trigger = no
 		}
```

Comment:
```suggestion
		NOT = { has_trait = infirm }
		age_ranked_health_vulnerability_threshold_trigger = yes
```

`NOT` for several triggers doesnt work as `NOT = { AND = {`. Paradox moment. You should make either `NOR` or `NAND`, or separate like i suggested.

Reply:
ok, i dont think it makes much of a difference but i am going to keep your suggestion because the double negative is less readable code.

Reply:
ok, this is more readable anyway